docs/doc/12-load-data/00-stage/00-whystage.md

```diff
@@ -57,6 +57,6 @@ Unlike a user stage, which is automatically created for each user and cannot be
 
 ### External Stage
 
-An external stage in Databend stores data files outside of the storage backend specified in databend-query.toml. When you create an external stage using the [CREATE STAGE](../../14-sql-commands/00-ddl/40-stage/01-ddl-create-stage.md) command, you'll need to specify the stage location where the data files are stored. The data can be stored on cloud storage services such as AWS S3, and Google Cloud Storage.
+An external stage in Databend stores data files outside of the storage backend specified in databend-query.toml. When you create an external stage using the [CREATE STAGE](https://databend.rs/doc/sql-commands/ddl/stage/ddl-create-stage) command, you'll need to specify the stage location where the data files are stored. The data can be stored on cloud storage services such as AWS S3, and Google Cloud Storage.
```

Comment:
Why change to this style link? We can check it's a dead links or not when yarn build.